### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by henrikhodne**
The last release was August 31st, 2015. There have been a lot of commits and complete restructures (move to gb and back) that makes the diffs hard to read, so I didn't backfill the changelog apart from the version numbers and release dates.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```
Comment (henrikhodne) [inline CHANGELOG.md]: Should we commit to adhering to Semantic Versioning?

Comment (meatballhat) [inline CHANGELOG.md]: I'm in favor.  Reason(s) not to?

Comment (henrikhodne) [inline CHANGELOG.md]: As far as I know, we don't commit to SemVer in any other projects (at least not any of the Blue ones).

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline CHANGELOG.md]: I think that is generally a good idea, but it sort of only makes sense if you have defined a public API.

I'm not sure what we would consider the public API of jupiter-brain. Perhaps the CLI + env vars? However, it gets more nuanced. Does removing an env var require a major release?

Comment (henrikhodne) [inline CHANGELOG.md]: We also don't really define a public API right now, which is a requirement for SemVer.

Comment (henrikhodne) [inline CHANGELOG.md]: I suppose I could just dump the HTTP API spec in the README.

Comment (meatballhat) [inline CHANGELOG.md]: Welp, for whatever such a commitment might mean, I'm in favor.  I don't want to invent our own system or assign our own semantics :grin:

Comment (henrikhodne) [inline CHANGELOG.md]: Okay, SemVer it is, I'll submit a PR to define the API in the README (just dump the current one).

**Pull request merged**